tests/integration/span_events/test_span_events_are_created_upon_caught_exception.php

```diff
@@ -6,13 +6,15 @@
 
 /*DESCRIPTION
```

Comment:
Will the test name be updated to `tests/integration/span_events/test_span_events_are_created_upon_uncaught_handled_exception.php`
in this or another PR?

Reply:
Initially we discussed that refactoring renaming the tests will be its own PR. However, since then, test renames in `jit` subtree became part of #715. Therefore, I think it makes sense for this PR to complement #715, and have test renames in this PR as well.

Reply:
Files renamed in 13fd3ffd7dff3075a00958387c08d6340c2361a1.